apps/webpack.js

```diff
@@ -5,23 +5,19 @@ var LiveReloadPlugin = require('webpack-livereload-plugin');
 var envConstants = require('./envConstants');
 var WebpackNotifierPlugin = require('webpack-notifier');
 
-// Certain packages ship in ES6 and need to be transpiled for our purposes -
-// especially for tests, which run on PhantomJS with _zero_ ES6 support.
+// Certain packages ship in ES6 and need to be transpiled for our purposes.
```

Comment:
Do we know what these purposes are? I think it includes (1) IE support, but also (2) extract common sub-dependencies from these dependencies so that we do not include them twice in our js bundles. Does that match your understanding?

Reply:
I thought it was only (1), but it makes sense that (2) would also be happening.  Do you think that matters for these test only packages I'm removing from the list, or are you wanting us to better capture the reason for the packages left behind here?

Reply:
I don't think it matters for the test-only packages, only for the ones you're leaving. I think it would be worth noting the reasons we might still have anything in this list -- listing the performance benefit as a "possible" reason would be fine :)